Vital-Frontend/src/app/toasts/toast.component.ts

```diff
@@ -11,7 +11,7 @@ import {animate, style, transition, trigger} from "@angular/animations";
            max-w-[300px] w-[200px] z-50 min-h-[100px] max-h-[150px]
            overflow-ellipsis
            lg:w-[300px]"
-           [class.bg-secondary]="toast.type === 'success'" [class.bg-error]="toast.type === 'error'"
+           [class.bg-success]="toast.type === 'success'" [class.bg-error]="toast.type === 'error'"
```

Comment:
the success colour feels very disruptive to our very harmonic theme lmao

Reply:
i have no strong opinion, i will change it